### PR TITLE
Update all monitor assets to v2 schema

### PR DESCRIPTION
**Pull request by yaodingyd**
### What does this PR do?

This PR updates all monitors assets to be in schema v2. There are no new data introduced, only backfilling the new meta fields.

The v2 schema is:

Name | Type | Required | Description
-- | -- | -- | --
version | number | true | Version of the schema used to encode the recommended monitor. Mainly used to make sure we can safely iterate on the data model without breaking the front-end.
created_at | string (yyyy-mm-dd) | true | When the recommended monitor was first created.
last_updated_at | string (yyyy-mm-dd) | true | When the recommended monitor was last updated.
tags | string[] | false | List of tags used to power the search capabilities.Allowed keys for now are integration, product and stack_component.
title | string | true | Title of the use-case as seen in the Datadog UI.
description | markdown | true | High-level description of the use-case and why users should consider leveraging it.
definition | Monitor definition | true | The monitor definition as it is exported today from the UI (even though it can be templated).

Details changes are:
1. Adding `version:2` to all monitor assets
2. `created_at`/`last_updated_at` are inferred with git commit history.
3. `tags` is only `integration:{app_id}` This is meta tags used in searching recommended monitors, which are different from the `tags` from monitor definition that are part of monitor.
4. The rest are copied from original schema, and monitor definition is moved to `definition`
5. Change folder from "recommended_monitors" to "monitors"

### Motivation

What inspired you to submit this pull request?

### Review checklist

- [ ] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [ ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

File: algorithmia/assets/monitors/algorithm_duration.json [hunk 1]
```diff
@@ -1,28 +1,35 @@
...
-	"recommended_monitor_metadata": {
-		"description": "Get notified when algorithms are taking more that 10 seconds to execute in Algorithmia."
-	}
-}
+  "version": 2,
+  "created_at": "2020-10-26",
+  "last_updated_at": "2021-02-16",
+  "title": "Excessive Algorithm Runtime",
+  "tags": [
+    "integration:algorithmia"
```
Comment (nmuesch) [inline algorithmia/assets/monitors/algorithm_duration.json:7]: Is this tags key needed twice (I see it once here and once again inside the `definition` key)

**Review by nmuesch — APPROVED** [review]

**Comment by nmuesch** [comment]
You can ignore the `config model` CI errors, those should be run and fixed when actual integration code is being updated.

**Review by hestonhoffman — APPROVED** [review]

**Comment by eho1307** [comment]
Hi Technology Partners! To add more context to this PR, our monitors team is updating the schema of recommended monitors and this change migrates your monitors to support the new schema. No new data is being introduced. Please reach out if you have any questions. We plan to merge this PR next Monday, August 21st.

cc @nbk96f1 @koverholt @chenyingtz @srinivas-bitla @signalsciences

File: emnify/assets/monitors/emnify_data_usage_forecast.json [hunk 2]
```diff
@@ -1,24 +1,33 @@
...
-    "new_host_delay": 300,
-    "notify_no_data": false,
-    "renotify_interval": 0
-  },
-  "priority": null,
-  "restricted_roles": null,
-  "recommended_monitor_metadata": {
-    "description": "Notify your team when forecasted daily use is higher than threshold."
+    "priority": null,
+    "query": "max(next_2w):forecast(sum:emnify.endpoint.volume{*}.as_count().rollup(sum, 86400), 'linear', 2, model='reactive', interval='60m', history='1mo') \u003e= 1000000000",
```
Comment (nmuesch) [inline emnify/assets/monitors/emnify_data_usage_forecast.json:26]: Sanity check: does the `<` and `>` characters in the query really need to be encoded?

**Review by nmuesch — APPROVED** [review]

**Pull request merged**